lib/report-generator.js

```diff
@@ -0,0 +1,95 @@
+var fs = require('fs-extra'),
+    path = require('path');
+
+/**
+ * The directory of the visual report
+ *
+ * @type {String}
+ * @const
+ * @default
+ */
+var DIRECTORY_PATH = 'visual-report';
+
+/**
+ * The directory of the static files
+ *
+ * @type {String}
+ * @const
+ * @default
+ */
+var STATIC_DIRECTORY = 'statics';
+
+/**
+ * The path of the data file
+ *
+ * @type {String}
+ * @const
+ * @default
+ */
+var DATA_PATH = 'data.js';
+
+/**
+ * The root directory
+ *
+ * @type {String}
+ */
+var ROOT_DIRECTORY;
```

Comment:
This is not a constant (you're changing its value below), therefore you need to define it in the block where it's being used. Only constants sit at the top of a module.